.github/workflows/main.yml

```diff
@@ -0,0 +1,107 @@
+name: CmdStanPy
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  cmdstanpy:
+    name: tests and publish
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies (python)
+        run: |
+          pip install --upgrade pip wheel
+          pip install -r requirements.txt
+          pip install -r requirements-test.txt
+          pip install codecov
+
+      - name: Run flake8 and pylint
+        run: |
+          mkdir run_tests
+          cd run_tests
+          flake8 --extend-ignore=E127,E201,E202,E203,E231,E252,E266,E402,E999,F841,W503,W605 --max-line-length=80 ../cmdstanpy ../test
+          pylint -v --rcfile=../.pylintrc ../cmdstanpy ../test
+
+      - name: Build wheel
+        run: |
+          python setup.py bdist_wheel
+
+      - name: Install wheel (Linux, macOS)
+        if: matrix.os != 'windows-latest'
+        run: |
+          pip install dist/*.whl
+
+      - name: Install wheel (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          $whl = Get-ChildItem -Path dist -Filter *.whl | Select-Object -First 1
+          pip install "$whl"
+
+      - name: Show libraries
+        run: |
+          python -m pip freeze
+
+      - name: Install CmdStan (Linux, macOS)
+        if: matrix.os != 'windows-latest'
+        run: |
+          python -m cmdstanpy.install_cmdstan --progress
```

Comment:
is flag --progress necessary?

Reply:
Just nice to have. It is so fast in CI that in normal case, devs don't need to care about it.